File: CSharpSample_DLL/SIPIMSample/Form2.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SIPIMSample
{
    public partial class Form2 : Form
    {
        private bool Accept = true;
        public bool GetAccept()
        {
            return Accept;
        }
        public Form2()
        {
            InitializeComponent();
        }
        public void SetContactName(string ContactName)
        {
            lblFrom.Text = ContactName;
        }
        public void SetSubject(string Subject)
        {
            ;
        }

        private void button1_Click(object sender, EventArgs e)
        {
            Accept = true;
            Close();
        }

        private void button2_Click(object sender, EventArgs e)
        {
            Accept = false;
            Close();
        }
    }
}

File: VBSample/SIPIMSample/Form2.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Public Partial Class Form2
	Inherits Form
	Private Accept As Boolean = True
	Public Function GetAccept() As Boolean
		Return Accept
	End Function
	Public Sub New()
		InitializeComponent()
	End Sub
	Public Sub SetContactName(ContactName As String)
		lblFrom.Text = ContactName
	End Sub
	Public Sub SetSubject(Subject As String)
		

	End Sub

	Private Sub button1_Click(sender As Object, e As EventArgs)
		Accept = True
		Close()
	End Sub

	Private Sub button2_Click(sender As Object, e As EventArgs)
		Accept = False
		Close()
	End Sub
End Class
